docs/myst.yml

```diff
@@ -1,5 +1,7 @@
 # See docs at: https://mystmd.org/guide/frontmatter
 version: 1
+extends:
+  - https://raw.githubusercontent.com/jupyter-book/meta/refs/heads/main/myst.yml
```

Comment:
This broke the `docs/` link at the top because the landing page doesn't have access to the sidebar of the TOC. If we want `docs/` to point to `next.jupyterbook.org` then we should define a different way for users to access the TOC of the documentation.

It also made me realize that when that sidebar is gone, then all the navbar items are gone too on mobile, because the hamburgur menu is gone with no sidebr, even though the navbar items are also in the same hamburger menu...

Reply:
I've made the `/docs` link an absolute URL to `next.jupyterbook.org`. Am I misunderstanding you -- do you want `docs` to point to move away from the landing page onto some content? 

I don't love the present view, but you can navigate away from the landing page via the inline links or bottom buttons. My feeling is that we might want to make it possible to have a hide-able sidebar even at full screen, so that the user sees a collapsed burger menu by default? That would facilitate landing pages without forcing us to use narrow views, or drop the sidebar.

> It also made me realize that when that sidebar is gone, then all the navbar items are gone too on mobile, because the hamburgur menu is gone with no sidebr, even though the navbar items are also in the same hamburger menu..

Yes, this is a problem with losing the sidebar. I think I suggested restoring it (for now) somewhere, but I've since had the thought about the theme above.

Reply:
I think it'll confuse people if they hit a landing page (jupyterbook.org), then click on "docs" at the top, and are just redirected to the same page. The landing page is going to feel like a different website from the "docs", which is why I'd made `docs/` point to the first page of the docs, rather than the landing page.

So the way I thought about it, `next.jupyterbook.org` is the future landing page for Jupyter Book, not the first page of the docs.

That said, I suggest you play around with different UX here and I'm happy to provide feedback on what feels intuitive.